### PR TITLE
Clarify wanted context

### PR DESCRIPTION
**Pull request by CassandraGoose**
Let me know if I'm misunderstanding this exercise!

**What:**
 I adjusted the wording in the README.md within the context folder and added some clarifying gifs. 

**Why:** 

Let me know if you can think of a better way to clarify this for students. 

- As a student, I would be confused about why what I see in my terminal is different than what I am told to answer and would probably give up and just copy and paste my answers from the test without actually learning anything. Maybe it would also be okay to add a link to this https://developer.mozilla.org/en-US/docs/Glossary/Global_object?
- As an instructor, I completely missed the sentence that said to "Imagine the code within the method is in it's own separate JavaScript file." but even after finding that statement I did not understand why we were calling it the 'global _window_ object' instead of just 'global object'. 

Basically, I believe it's misleading to have the students answer 'global window object' when that doesn't exist where they are running the JS _and_ it's confusing to have students pretend the functions are being run in a context that they are not actually being run.

**Notes:**
- I have multiple commits due to GitHub's markdown whitelist ignoring style tags in images and needing to see it to know if it worked... Please squash.

### CODE REVIEW

**Comment by wvmitchell** [comment]
I think this is great, giving them the visual cues is superhelpful. I'd say go ahead and squash then commit!

**Comment by wvmitchell** [comment]
It's also worth noting that the 'context' JSFun exercises are the most often skipped. I'll be interested to see if more students take advantage of them with the added clarity.
